org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/LoggerTest.java

```diff
@@ -0,0 +1,166 @@
+package org.eclipse.lsp4xml.utils;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+
+import org.eclipse.lsp4j.InitializeParams;
+import org.eclipse.lsp4j.MessageActionItem;
+import org.eclipse.lsp4j.MessageParams;
+import org.eclipse.lsp4j.MessageType;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.PublishDiagnosticsParams;
+import org.eclipse.lsp4j.ShowMessageRequestParams;
+import org.eclipse.lsp4j.services.LanguageClient;
+import org.eclipse.lsp4xml.XMLLanguageServer;
+import org.eclipse.lsp4xml.internal.parser.XMLParser;
+import org.eclipse.lsp4xml.model.XMLDocument;
+import org.eclipse.lsp4xml.services.XMLLanguageService;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+/**
+ * LoggerTest
+ */
+public class LoggerTest {
+  private String path = "src/test/java/org/eclipse/lsp4xml/utils/logTestFiles/testLogFile.log";
+  private MockLanguageClient mockLanguageClient;
+  private File logFile;
+  private XMLLanguageService languageService;
```

Comment:
you don't need XMLLanguageService nor XMLDocument

Reply:
create handle to Logger: 
Logger logger = Logger.getLogger(LoggerTest.class.getName());